setup.py

```diff
@@ -54,6 +54,7 @@ def get_fastseq_version():
         "packaging",
         "torch>=1.4.0",
         "pytorch-transformers==1.0.0",
+        "transformers==3.0.2"
```

Comment:
I think it is already handled in extras_require

Reply:
I added this because the CI test reported an error of `No module named 'transformers'`. 
https://github.com/microsoft/fastseq/pull/80/checks?sha=2f3ef72e81fefd4a0e37f3ca29b8c5e1ba7367da

Reply:
@fuliucansheng  As FastSeq supports both FairSeq and HuggingFace-Transformers, these two packages are added as the [extras_require](https://github.com/microsoft/fastseq/blob/main/setup.py#L18-L21). 

To handle the above issue, could you please handle it like [here](https://github.com/microsoft/fastseq/blob/main/fastseq/models/prophetnet_fs/__init__.py#L12-L23), which throws a warning message instead of an error when `transformers` is not installed.

Reply:
And this error is strange "No module named 'transformers'". It should be already installed in CI. Any idea?

Reply:
@JiushengChen The cause is [here](https://github.com/microsoft/fastseq/blob/main/tests/run_fairseq_tests.py#L26-L32): the unit tests of FairSeq run in another virtual environment, which does not install transformers.

Reply:
@feihugis , what is the solution?

Reply:
The solution is to throw a warning [here](https://github.com/microsoft/fastseq/pull/80/files#diff-d722765538278d903ef4b391db1a989ac3ba213f7a364b57e1ffba3ed1fcb14aR12-R16).